sketchybar/icons.sh

```
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+APPLE=""
```

Shellcheck (0.11.0)

[warning] APPLE appears unused. Verify use (or export if used externally).

(SC2034)